modules/afsmtp/afsmtp.c

```diff
@@ -626,6 +666,8 @@ afsmtp_dd_init(LogPipe *s)
     return FALSE;
 
   g_list_foreach(self->headers, (GFunc)afsmtp_dd_init_header, cfg);
+  g_list_foreach(self->rcpt_tos, (GFunc)afsmtp_dd_init_recipient, cfg);
+
```

Comment:
This seems to be an independent change to me, and as such should go to a separate patch